### PR TITLE
HDDS-5913. Avoid integer overflow when setting dfs.container.ratis.lo…

### PR DESCRIPTION
**Pull request by ChenSammi**
https://issues.apache.org/jira/browse/HDDS-5913

### CODE REVIEW

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/XceiverServerRatis.java [hunk 1]
```diff
@@ -312,7 +312,8 @@ private RaftProperties newRaftProperties() {
         StorageUnit.BYTES);
     RaftServerConfigKeys.Log.setQueueElementLimit(
         properties, logQueueNumElements);
-    RaftServerConfigKeys.Log.setQueueByteLimit(properties, logQueueByteLimit);
+    RaftServerConfigKeys.Log.setQueueByteLimit(properties,
+        SizeInBytes.valueOf(logQueueByteLimit));
```
Comment (ayushtkn) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/XceiverServerRatis.java:316]: How does this prevent integer overflow of ``logQueueByteLimit``, This value is type casted into `(int)` above
```
    final int logQueueByteLimit = (int) conf.getStorageSize(
        OzoneConfigKeys.DFS_CONTAINER_RATIS_LOG_QUEUE_BYTE_LIMIT,
        OzoneConfigKeys.DFS_CONTAINER_RATIS_LOG_QUEUE_BYTE_LIMIT_DEFAULT,
        StorageUnit.BYTES);
```
Can you help share some pointers

Comment (ChenSammi) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/XceiverServerRatis.java:316]: You are right.  This patch is seperated from one of my interval big patch which mixes several fixes.  The cast to (long) line is missed here.

**Review by ayushtkn — APPROVED** [review]
Changes LGTM.
Give a check to the Test failures, if related

**Review by adoroszlai — APPROVED** [review]
Thanks @ChenSammi for the fix.  Would you mind adding a unit test to verify value in `RaftProperties`?

**Comment by adoroszlai** [comment]
Thanks @ChenSammi for the patch, @ayushtkn for the review.  I guess we can add the unit test later.

**Pull request merged**